<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QDialog" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>497</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="contextMenuPolicy">
      <enum>Qt::ActionsContextMenu</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#widget{
	border-top-left-radius:10px;
	border-bottom-left-radius:10px;
	border-top-right-radius:10px;
	border-bottom-right-radius:10px;
	border: 1px solid  #3daee9;
	background-color: #19232D;
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="lblTitle">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border-top-left-radius:10px;
border-top-right-radius:10px;
border: 1px solid  #3daee9;
border-bottom: none;
</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignHCenter|Qt::AlignTop</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="leftMargin">
         <number>9</number>
        </property>
        <property name="topMargin">
         <number>9</number>
        </property>
        <property name="rightMargin">
         <number>9</number>
        </property>
        <property name="bottomMargin">
         <number>9</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,0">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <layout class="QVBoxLayout" name="iconHolder">
              <property name="spacing">
               <number>0</number>
              </property>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QScrollArea" name="scrollArea">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>447</width>
               <height>134</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="lblMessage">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>TextLabel</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Options</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <property name="spacing">
            <number>3</number>
           </property>
           <property name="leftMargin">
            <number>3</number>
           </property>
           <property name="topMargin">
            <number>3</number>
           </property>
           <property name="rightMargin">
            <number>3</number>
           </property>
           <property name="bottomMargin">
            <number>3</number>
           </property>
           <item>
            <layout class="QGridLayout" name="gridLayout">
             <property name="leftMargin">
              <number>6</number>
             </property>
             <property name="topMargin">
              <number>6</number>
             </property>
             <property name="rightMargin">
              <number>6</number>
             </property>
             <property name="bottomMargin">
              <number>6</number>
             </property>
             <item row="0" column="0">
              <widget class="QPushButton" name="pushButton_quote">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>220</width>
                 <height>40</height>
                </size>
               </property>
               <property name="toolTip">
                <string>Will generate and save a quote to the Quote directory</string>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton#pushButton_quote{
	color: #ffffff;
    background-color: #3daee9;
    border: 0.01em solid #3daee9;
    border-radius: 8px;
}
QPushButton#pushButton_quote:checked{
    background-color: #3daee9;
    border: 0.01em solid #3daee9;
}
QPushButton#pushButton_quote:hover{
    background-color: #48b6ed;
    border-color: #48b6ed;
}
QPushButton#pushButton_quote:pressed{
    background-color: #2b92c5;
    border-color: #2b92c5;
}

QPushButton#pushButton_quote:!checked {
    background-color: rgba(71, 71, 71, 130);
    border: 0.01em solid rgba(76, 76, 76, 130);
    border-radius: 8px;
    color: grey;
}

QPushButton:hover#pushButton_quote:!checked {
    background-color: rgba(76, 76, 76, 130);
    border: 0.01em solid rgba(71, 76, 88, 130);
}

QPushButton:pressed#pushButton_quote:!checked {
    background-color: rgba(39, 39, 39, 130);
    border: 0.01em solid rgba(47, 50, 57, 130);
}</string>
               </property>
               <property name="text">
                <string>Generate Quote</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QPushButton" name="pushButton_update_inventory">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>220</width>
                 <height>39</height>
                </size>
               </property>
               <property name="toolTip">
                <string>Will add/update parts in inventory.</string>
               </property>
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton#pushButton_update_inventory{
	color: #ffffff;
    background-color: #3daee9;
    border: 0.01em solid #3daee9;
    border-radius: 8px;
}
QPushButton#pushButton_update_inventory:checked{
    background-color: #3daee9;
    border: 0.01em solid #3daee9;
}
QPushButton#pushButton_update_inventory:hover{
    background-color: #48b6ed;
    border-color: #48b6ed;
}
QPushButton#pushButton_update_inventory:pressed{
    background-color: #2b92c5;
    border-color: #2b92c5;
}

QPushButton#pushButton_update_inventory:!checked {
    background-color: rgba(71, 71, 71, 130);
    border: 0.01em solid rgba(76, 76, 76, 130);
    border-radius: 8px;
    color: grey;
}

QPushButton:hover#pushButton_update_inventory:!checked {
    background-color: rgba(76, 76, 76, 130);
    border: 0.01em solid rgba(71, 76, 88, 130);
}

QPushButton:pressed#pushButton_update_inventory:!checked {
    background-color: rgba(39, 39, 39, 130);
    border: 0.01em solid rgba(47, 50, 57, 130);
}</string>
               </property>
               <property name="text">
                <string>Add Parts to Inventory</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QPushButton" name="pushButton_workorder">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>220</width>
                 <height>40</height>
                </size>
               </property>
               <property name="toolTip">
                <string>Will generate and save a workorder to the Workorder directory</string>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton#pushButton_workorder{
	color: #ffffff;
    background-color: #3daee9;
    border: 0.01em solid #3daee9;
    border-radius: 8px;
}
QPushButton#pushButton_workorder:checked{
    background-color: #3daee9;
    border: 0.01em solid #3daee9;
}
QPushButton#pushButton_workorder:hover{
    background-color: #48b6ed;
    border-color: #48b6ed;
}
QPushButton#pushButton_workorder:pressed{
    background-color: #2b92c5;
    border-color: #2b92c5;
}

QPushButton#pushButton_workorder:!checked {
    background-color: rgba(71, 71, 71, 130);
    border: 0.01em solid rgba(76, 76, 76, 130);
    border-radius: 8px;
    color: grey;
}

QPushButton:hover#pushButton_workorder:!checked {
    background-color: rgba(76, 76, 76, 130);
    border: 0.01em solid rgba(71, 76, 88, 130);
}

QPushButton:pressed#pushButton_workorder:!checked {
    background-color: rgba(39, 39, 39, 130);
    border: 0.01em solid rgba(47, 50, 57, 130);
}</string>
               </property>
               <property name="text">
                <string>Generate Workorder</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QPushButton" name="pushButton_packingslip">
               <property name="minimumSize">
                <size>
                 <width>220</width>
                 <height>40</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton#pushButton_packingslip{
	color: #ffffff;
    background-color: #3daee9;
    border: 0.01em solid #3daee9;
    border-radius: 8px;
}
QPushButton#pushButton_packingslip:checked{
    background-color: #3daee9;
    border: 0.01em solid #3daee9;
}
QPushButton#pushButton_packingslip:hover{
    background-color: #48b6ed;
    border-color: #48b6ed;
}
QPushButton#pushButton_packingslip:pressed{
    background-color: #2b92c5;
    border-color: #2b92c5;
}

QPushButton#pushButton_packingslip:!checked {
    background-color: rgba(71, 71, 71, 130);
    border: 0.01em solid rgba(76, 76, 76, 130);
    border-radius: 8px;
    color: grey;
}

QPushButton:hover#pushButton_packingslip:!checked {
    background-color: rgba(76, 76, 76, 130);
    border: 0.01em solid rgba(71, 76, 88, 130);
}

QPushButton:pressed#pushButton_packingslip:!checked {
    background-color: rgba(39, 39, 39, 130);
    border: 0.01em solid rgba(47, 50, 57, 130);
}</string>
               </property>
               <property name="text">
                <string>Generate Packing Slip</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="buttonsLayout"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
